lib/miq_automation_engine/service_models/miq_ae_service_manageiq-providers-ansible_tower-configuration_manager-job.rb

```diff
@@ -2,6 +2,7 @@ module MiqAeMethodService
   class MiqAeServiceManageIQ_Providers_AnsibleTower_ConfigurationManager_Job < MiqAeServiceOrchestrationStack
     expose :job_template, :association => true
     expose :refresh_ems
+    expose :stdout, :method => :raw_stdout
```

Comment:
@mkanoor Ok. I will use the same name. The naming convention is to use prefix `raw_` if the value comes directly from the provider.